packages/@webex/internal-plugin-mercury/src/socket/socket-base.js

```diff
@@ -489,11 +489,21 @@ export default class Socket extends EventEmitter {
       }
     };
 
+    const calculateLatency = (pingTimestamp) => {
+      const now = new Date().getTime();
+      const latency = now - pingTimestamp;
+
+      this.logger.info(`socket,${this._domain}: latency: `, latency);
```

Comment:
is this log going to show up in the logs every 10s? seems a bit excessive, it will fill the log for longer meetings like 1h

Reply:
hm, it's every 19 sec but i think i will replace info with debug